content/applications/inventory_and_mrp/inventory/management/delivery/dropshipping.rst

```diff
@@ -1,109 +1,80 @@
-        One reason is that you'll always make smaller
-        margins on items that are drop-shipped, so you should keep it
-        only for items that take up a lot of space in your warehouse.
-
--  Drop-shipping is best for niche products.
-       Chances are that products that are in high demand are being offered by large suppliers at a
-       fraction of the price you'll be able to charge, so using a more
-       costly shipping method won't be financially rewarding. But if
-       your product is unique, then it makes sense!
-
--  To protect your customers from bad experiences.
-       Test drop-shipping companies for yourself beforehand and list the best ones.
-
--  Make sure time is not against you.
-       Drop-shipping should take a reasonable amount of time and surely not more than it would have taken you to handle it all by yourself. It's also nice to be able
-       to provide your customers with a tracking number.
-
--  Items have to be available from your supplier.
-       It's good to know if
-       the product you're selling is available upstream. If you don't
-       have that information, inform your customers that you don't hold
-       the item in stock and that it's subject to availability from a
-       third party.
+=============================================================
+Use dropshipping to ship directly from suppliers to customers
+=============================================================
 
-.. tip::
-      For more information and insights about Drop-Shipping, you can read our
-      blog: `*What is drop-shipping and how to use
-      it* <https://www.odoo.com/blog/business-hacks-1/post/what-is-drop-shipping-and-how-to-use-it-250>`__.
+Dropshipping is an order fulfillment strategy that allows sellers to have items shipped directly
+from suppliers to customers. Normally, a seller purchases a product from a supplier, stores it in
+their inventory, and ships it to the end customer once an order is placed. With dropshipping, the
+supplier is responsible for storing and shipping the item. This benefits the seller by reducing
+inventory costs, including the price of operating warehouses.
+
+Configure products to be dropshipped
+====================================
 
-Configure Drop-Shipping
-=======================
+To use dropshipping as a fulfillment strategy, navigate to the :guilabel:`Purchase` app and
+select :menuselection:`Configuration --> Settings`. Under the :guilabel:`Logistics` heading, click
+the :guilabel:`Dropshipping` checkbox.
 
-Activate the functionality in the *Purchase* application by going to :menuselection:`Configuration
---> Settings`.
+Next, go to the :guilabel:`Sales` app, click :menuselection:`Products --> Products` and choose an
+existing product or select :guilabel:`Create` to configure a new one. On the product page, make sure
+that the :guilabel:`Can be Sold` and :guilabel:`Can be Purchased` checkboxes are enabled.
```

Comment:
These are fine, so more of a FYI: 
Can use the guilabel as an action verb in a sentence just to cut down on wordiness 🧠 
```suggestion
existing product or :guilabel:`Create` a new one. On the product page, make sure that the
:guilabel:`Can be Sold` and :guilabel:`Can be Purchased` checkboxes are enabled.
```